openapi/src/openapi-src-v2.json

```diff
@@ -17,6 +17,180 @@
+                            },
+                            "example": {
+                                "data": {
+                                    "type": "questionnaires",
+                                    "attributes": {
+                                        "templateName": "sexual-assault",
+                                        "owner": {
+                                            "id": "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6",
+                                            "isAuthenticated": true
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "201": {
+                        "description": "Created",
+                        "content": {
+                            "application/vnd.api+json": {
+                                "schema": {
+                                    "$ref": "./json-schemas/api/_questionnaires/post/res/201.json"
+                                },
+                                "example": {
+                                    "data": {
+                                        "type": "questionnaires",
+                                        "id": "todo-p-some-id",
+                                        "attributes": {
+                                            "sections": {},
+                                            "routes": {}
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/BadRequest"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/UnauthorizedError"
+                    },
+                    "403": {
+                        "$ref": "#/components/responses/Forbidden"
+                    },
+                    "404": {
+                        "$ref": "#/components/responses/NotFound"
+                    }
+                }
+            }
+        }
+    },
+    "components": {
+        "securitySchemes": {
+            "bearerAuth": {
+                "type": "http",
+                "scheme": "bearer",
+                "bearerFormat": "JWT"
+            }
+        },
+        "parameters": {
+            "apiVersion": {
+                "name": "Dcs-Api-Version",
+                "in": "header",
+                "required": true,
+                "description": "The date based version of the requested API",
+                "schema": {
+                    "$ref": "./json-schemas/models/definitions/api-version.json"
+                },
+                "example": "2027-05-17"
```

Comment:
😢

Reply:
@BarryPiccinni can you update this one too. Swagger UI uses it as a default value when trying out endpoints